internal/pkg/utils/testproject/project.go

```diff
@@ -517,7 +533,7 @@ func (p *Project) logEnvs() {
 func (p *Project) logf(format string, a ...interface{}) {
 	if testhelper.TestIsVerbose() {
 		seconds := float64(time.Since(p.initStartedAt).Milliseconds()) / 1000
-		a = append([]interface{}{p.ID(), p.t.Name(), seconds}, a...)
-		p.t.Logf("TestProject[%d][%s][%05.2fs]: "+format, a...)
+		a = append([]interface{}{p.ID(), seconds}, a...)
+		fmt.Printf("TestProject[%d][%05.2fs]: "+format, a...)
```

Comment:
Tady si nejsem jistej jestli je to ok a případně jestli jde jméno testu za něco vyměnit.

Reply:
hmm, no to logovanie sa zobrazovalo pri testoch s `TEST_VERBOSE=true`, ono sa to celkom moze hodit.

Moze sa to vyriesit tak, ze do `Project` sa prida field `logFn func(format string, a ...interface{})`, 
a ta sa nastavi, iba v `GetTestProjectForTest`, a tam sa presunie tento kod, ... a tu by sa uz ba kontrolovalo `p.logFn != nil`.

Ak chces, tak v `GetTestProject` mozes mat default implementaciu s `Printf`.
Ale to logovanie v testoch mi pride dobre zachovat, najma ak bezi viac testov paralelne, tak ten nazov testu tam potrebujeme.

Reply:
Jo, určitě jsem pro to zachovat, jen jsem si nevěděl rady jak to udělat. Tady to předávání funkcí v property struktur ještě nemám úplně zažitý. 😄 Mimo testy to asi ani potřeba není, tak jsem to nechal jen v tom `GetTestProjectForTest`: https://github.com/keboola/keboola-as-code/pull/752/commits/d6a5acfb0dc98b3d76b793a44ad3d214e967b4cc

Reply:
:+1: